cli/cmd/cluster.go

```diff
@@ -970,6 +973,63 @@ func refreshCachedClusterConfig(awsClient aws.Client, accessConfig *clusterconfi
 	return *refreshedClusterConfig
 }
 
+func scaleNodeGroup(clusterConfig clusterconfig.Config) (clusterconfig.Config, error) {
```

Comment:
Yep, it can be misleading.

Reply:
Just renamed the function and passed in the flags as arguments to the function. I don't necessarily think we should attach it to the `clusterconfig.Config` struct - that's because the errors are about the CLI flags. We can talk about this.